webpack.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+    optimization: {
+      minimize: false,
```

Comment:
:+1: if it _doesn't_ OoM without this anymore, that would be rad, but we do what we gotta.

Reply:
It actually doesn't OoM without it, turns out issues was elsewhere. Removing this config now.